js/components/controlButton.js

```diff
@@ -201,49 +209,51 @@ var ControlButton = createReactClass({
    * @param {type} key An id (usually the focusId prop) that identifies the button whose tooltip alignment we want to determine
    * @return {string} A value from CONSTANTS.TOOLTIP_ALIGNMENT which represents the tooltip alignment
    */
-  getTooltipAlignment: function(key) {
+  getTooltipAlignment(key) {
     if (typeof this.props.getTooltipAlignment === 'function') {
       return this.props.getTooltipAlignment(key);
     } else {
       return CONSTANTS.TOOLTIP_ALIGNMENT.CENTER;
     }
-  },
+  }
 
   /**
    * Handler for the mouseenter event.
    * @private
    * @param {Event} The mouseenter event object
    */
-  onMouseEnter: function(event) {
+  onMouseEnter(event) {
     this.highlight(event);
 
     if (typeof this.props.onMouseOver === 'function') {
       this.props.onMouseOver(event);
     }
-  },
+  }
 
   /**
    * Handler for the mouseleave event.
    * @private
    * @param {Event} The mouseleave event object
    */
-  onMouseLeave: function(event) {
+  onMouseLeave(event) {
     this.removeHighlight(event);
 
     if (typeof this.props.onMouseOut === 'function') {
       this.props.onMouseOut(event);
     }
-  },
+  }
+
+  render() {
+    const Component = this.getButtonComponent();
+    const className = classNames('oo-control-bar-item', this.props.className);
+    const iconStyles = this.getIconStyles();
+    const areTooltipsEnabled = this.areTooltipsEnabled();
 
-  render: function() {
-    var Component = this.getButtonComponent();
-    var className = classNames('oo-control-bar-item', this.props.className);
-    var iconStyles = this.getIconStyles();
-    var areTooltipsEnabled = this.areTooltipsEnabled();
+    let responsiveUiMultiplier, tooltipVerticalOffset;
```

Comment:
Can you put these are on two separate lines?

Reply:
Will update